### DISCUSSION TITLE
Ordering imports lint rule

### DISCUSSION THREAD

**Discussion by igortas** [rome/tools#4181] [Suggestions]
I think it's good if we have option to sort imports similar how `eslint-plugin-import` works.

`import react from 'react'`
`import _ from lodash`
`import { Box } from '@mui/material'`
`import './components/Button'`

or vise versa:

`import './components/Button'`
`import { Box } from '@mui/material'`
`import _ from lodash`
`import react from 'react'`

**Comment by ematipico** [comment]
This is something that Rome already supports via VSCode extension, and soon available via CLI too.

Check https://github.com/rome/tools/blob/main/editors/vscode/README.md and seach for important sorting
